src/constants/skills.js

```diff
@@ -343,6 +346,9 @@ const icons = {
   puppeteer: "https://i.ibb.co/Qk299CX/puppeteer.png",
   karma: "https://i.ibb.co/dbgh2DH/jasmine.png",
   jasmine: "https://i.ibb.co/55txF2S/karma.png",
+  gtk: "https://i.ibb.co/JyGnDrY/gtk.png",
```

Comment:
@MauricioHernanCabrera Please use SVG icons only. I will remove the present png icons too

Reply:
Okasss

Reply:
@rahuldkjain if you want, I can change the others skills of png to svg